fivox/compartmentLoader.cpp

```diff
@@ -49,8 +49,6 @@ class CompartmentLoader::Impl
         , _report( _config.getReportSource( params.getReport( )),
                    brion::MODE_READ, _target)
     {
-        _report.updateMapping( _target );
-
```

Comment:
It's called update, hence you can change the GIDs you want reportables for w/o throwing away the already opened report file. RTNeuron for instance can update the GIDs at runtime which are requested by the user or could be automatically if the visible neurons change for instance. Always creating a new compartment report object for that would be the other option.